crowbar_framework/app/views/barclamp/glance/_edit_attributes.html.haml

```diff
@@ -11,6 +11,9 @@
       %legend
         = t(".store_header")
       = select_field :default_store, :collection => :default_stores_for_glance
+      #default_store_info
```

Comment:
@bkutil you should already know that you don't need to add additional javascript for that functionality :)

```
= select_field :default_store, :collection => :default_stores_for_glance, "data-showit" => "file", "data-showit-target" => "#default_store_info", "data-showit-direct" => "true"
```

```
#default_store_info.alert.alert-info{ "data-show-for-clusters-only" => "true", "data-elements-path" => "glance-server" }
  = t('.store.file_ha_info')
```

For example: https://github.com/crowbar/barclamp-crowbar/blob/43d278b194d2e87562ecbea87a74f8ce9b1cc92b/crowbar_framework/app/views/nodes/list.html.haml#L115